.test-infra/jenkins/job_LoadTests_ParDo_Python.groovy

```diff
@@ -151,3 +151,35 @@ CronJobBuilder.cronJob('beam_LoadTests_Python_ParDo_Dataflow_Batch', 'H 13 * * *
   ]
   batchLoadTestJob(delegate, CommonTestProperties.TriggeringContext.POST_COMMIT)
 }
+
+def streamingLoadTestJob = { scope, triggeringContext ->
+  scope.description('Runs Python ParDo load tests on Dataflow runner in streaming mode')
+  commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 120)
+
+  def datasetName = loadTestsBuilder.getBigQueryDataset('load_test', triggeringContext)
+  for (testConfiguration in loadTestConfigurations("streaming", datasetName)) {
+    // Skipping case 2 in streaming because it timeouts. To be checked TODO: kkucharc
+    if(testConfiguration.title != "ParDo Python Load test: 2GB 100 byte records 200 times") {
```

Comment:
I am totally for refactoring those LoadTestJob methods into one. Thank you for suggestion.

As it comes to excluding the test, I am usually more for implicit pointing to the test that should be excluded than by pointing ex. index of array (maybe because that index can change in case of adding new test cases).